### PR TITLE
Fixed priorities of subscriber

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | none
| License | MIT
| Documentation PR | none

#### What's in this PR?

This PR fixes the priorities to generate correct routes with pagenumber.

### CODE REVIEW

File: Document/Subscriber/RoutableSubscriber.php [hunk 1]
```diff
@@ -65,7 +65,7 @@ public static function getSubscribedEvents()
     {
         return [
             Events::HYDRATE => [['handleHydrate', -500]],
-            Events::PERSIST => [['handleRouteUpdate', 1], ['handleRoute', 0]],
+            Events::PERSIST => [['handleRouteUpdate', -2048], ['handleRoute', -2049]],
```
Comment (alexander-schranz) [inline Document/Subscriber/RoutableSubscriber.php]: pls add some space between the priorities to allow someone to inject a validation or something

File: Document/Subscriber/RoutableSubscriber.php [hunk 2]
```diff
@@ -19,15 +19,19 @@
...
 use Sulu\Component\DocumentManager\Events;
+use Sulu\Component\DocumentManager\PropertyEncoder;
 use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 
 /**
  * Handles document-manager events to create/update/remove routes.
  */
 class RoutableSubscriber implements EventSubscriberInterface
 {
+    const FIELD = 'routePath';
```
Comment (alexander-schranz) [inline Document/Subscriber/RoutableSubscriber.php]: shouldn't we use a tag to mark the `routePath` and not hardcode its property name.

File: Document/ArticlePageViewObject.php [hunk 3]
```diff
@@ -44,6 +44,21 @@ class ArticlePageViewObject
...
 
     /**
+     * @var string
+     *
+     * @Property(
+     *     type="string",
+     *     options={
+     *         "fields":{
+     *            "raw":{"type":"string", "index":"not_analyzed"},
+     *            "value":{"type":"string"}
```
Comment (alexander-schranz) [inline Document/ArticlePageViewObject.php]: should we use the path_hierarchy tokenizer here?

Comment (wachterjohannes) [inline Document/ArticlePageViewObject.php]: is a BC break which needs recreating the whole index. but makes sense.

Comment (wachterjohannes) [inline Document/ArticlePageViewObject.php]: `path_hierachy` is only a tokenizer and we should avoid prepending config to elasticsearch-bundle.

See https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-pathhierarchy-tokenizer.html#_example_configuration_12

**Comment by wachterjohannes** [comment]
@alexander-schranz fixed

File: Document/ArticlePageViewObject.php (hunk 3 above)
Comment (wachterjohannes) [inline Document/ArticlePageViewObject.php]: i have created an issue for that problem https://github.com/sulu/SuluArticleBundle/issues/140

**Pull request merged**